### DISCUSSION TITLE
Ant Design Value Select doesn't allow searching

### DISCUSSION THREAD

**Discussion by A-EbrahimRSA** [react-querybuilder/react-querybuilder#826] [Ideas]
I've come to identify that the Ant Design Value Select for a field with a select type doesn't allow searching.

For example when looking at the Primary instrument value select on the demo page, majority of the styling libraries provide some searching functionality i.e. when typing A=>Accordion becomes selected. The Ant Design Value Select by default doesn't behave the same.

I think this would be a nice value add in the default behaviour.

**Comment by A-EbrahimRSA** [comment]
My current work around for this is a simple wrapper of the value editor to replicate the behaviour I've explained above

```javascript
const CustomValueSelector = (props: ValueEditorProps) => (
    <AntDValueEditor {...props} selectorComponent={selectorProps => <AntDValueSelector {...selectorProps} showSearch />} />
);

````

**Reply by jakeboone02** [reply]
This is a good point and something I considered turning on by default when I created the compatibility package.

I think rather than change the default behavior at this point, I would rather add a note similar to https://github.com/react-querybuilder/react-querybuilder/discussions/827#discussioncomment-11474262 on the package README.

> You can also replace _all_ value selectors, including the ones used by the value editor component, with the `controlElements.valueSelector` prop:
> 
> ```tsx
> const CustomValueSelector = (props: ValueSelectorProps) => (
>   <AntDValueSelector {...props} showSearch optionFilterProp='label' />
> );
> 
> export const App = () => {
>   return (
>     <QueryBuilder controlElements={{ valueSelector: CustomValueSelector }} />
>   );
> }
> ```
